google-compute-service.js

```diff
@@ -809,4 +809,17 @@ module.exports = class GoogleComputeService {
 
     return res;
   }
+
+  async listFirewallRules(params) {
+    const firewallsClient = new compute.FirewallsClient({ credentials: this.credentials });
+    const project = parsers.autocomplete(params.project) || this.projectId;
```

Comment:
Params parsing belogns to infrastructure layer, while this file is business logic/use case oriented.
I realize ther might be other such constructions in this file which require fixing, but let's not introduce new faults.

Please pass already parsed params to this class/method.

Reply:
Done